modules/mono/glue/GodotSharp/GodotSharp/Core/NativeInterop/NativeFuncs.cs

```diff
@@ -60,6 +60,7 @@ private partial struct UnmanagedCallbacks
             ref DebuggingUtils.godot_stack_info_vector p_stack_info_vector);
 
         internal static partial void godotsharp_internal_editor_file_system_update_file(in godot_string p_script_path);
+        internal static partial void godotsharp_internal_editor_file_system_update_files(ref godot_packed_string_array p_script_paths);
```

Comment:
We usually use `in` when the parameter is not going to be modified, although in practice this doesn't make a difference.

```suggestion
        internal static partial void godotsharp_internal_editor_file_system_update_files(in godot_packed_string_array p_script_paths);
```

Reply:
Oh nice, I was not able to use using because of the ref!